### PR TITLE
add code scheme placeholders

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Demog codes adapted from keep-II

### CODE REVIEW

File: code_schemes/age_category.json [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    },
+    {
+      "CodeID": "code-EC-3a704f5b",
+      "CodeType": "Meta",
+      "MetaCode": "exclusion_complaint",
+      "DisplayText": "meta: exclusion complaint",
+      "NumericValue": -100070,
+      "StringValue": "exclusion_complaint",
+      "VisibleInCoda": true
+    }
```
Comment (as2388) [inline code_schemes/age_category.json:213]: impact?

File: code_schemes/kakuma_household_language.json [hunk 2]
```diff
@@ -0,0 +1,215 @@
+{
+  "SchemeID": "Scheme-eccc5eda",
+  "Name": "Kakuma household language",
```
Comment (as2388) [inline code_schemes/kakuma_household_language.json:3]:
```suggestion
  "Name": "Kakuma Household Language",
```

File: code_schemes/kakuma_location.json [hunk 3]
```diff
@@ -0,0 +1,183 @@
+{
+  "SchemeID": "Scheme-d2a3de05",
+  "Name": "Kakuma location",
```
Comment (as2388) [inline code_schemes/kakuma_location.json:3]:
```suggestion
  "Name": "Kakuma Location",
```

File: code_schemes/kakuma_ws_correct_dataset.json [hunk 4]
```diff
@@ -0,0 +1,584 @@
...
+      "NumericValue": 10,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma leap s01e05"
+      ]
+    },
+    {
+      "CodeID": "code-ba352823",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma s02e01",
```
Comment (as2388) [inline code_schemes/kakuma_ws_correct_dataset.json:119]: I wonder if it's worth tweaking the display text to e.g. kakuma keep ii s02e01, just to make the difference super clear.

**Review by as2388 — APPROVED** [review]

File: code_schemes/age_category.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/age_category.json:276]: No really useful under age_category scheme but have updated all the schemes to include the latest meta codes including meta safeguarding.

File: code_schemes/kakuma_household_language.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/kakuma_household_language.json]: updated thanks

File: code_schemes/kakuma_location.json (hunk 3 above)
Comment (IsaackMwenda) [reply code_schemes/kakuma_location.json]: updated thanks

File: code_schemes/kakuma_ws_correct_dataset.json (hunk 4 above)
Comment (IsaackMwenda) [reply code_schemes/kakuma_ws_correct_dataset.json]: A good suggestion to reduce coding errors. Updated for DisplayText only for compatibility with older datasets.

**Pull request merged**